test/checks/keyboard/landmark-is-top-level.js

```diff
@@ -45,26 +52,27 @@ describe('main-is-top-level', function () {
 		navDiv.setAttribute('role','navigation');
 		fixture.appendChild(navDiv);
 		fixture.appendChild(mainDiv);
-		assert.isTrue(checks['main-is-top-level'].evaluate(mainDiv));
+		assert.isTrue(check.evaluate.call(checkContext, mainDiv));
+		assert.deepEqual(checkContext._data, { role: 'main' });
 	});
 	
 	it('should return true if main is in form landmark', function () {
```

Comment:
Some of these test names are now outdated given the nature of the changes therein.